src/utils/git.ts

```diff
@@ -17,7 +17,7 @@ export function parseGitUrl(maybeUrl: string) {
     url.startsWith(`https://${provider_domain}/`)
   ) {
     const [, owner, repo] =
-      url.match(new RegExp(`http[s]*://${provider_domain}/([^/]+)/([^/]+)/?.*$`)) || [];
+      url.match(new RegExp(`http[s]*://${provider_domain}/([^/]+)/([^/#]+)/?.*$`)) || [];
```

Comment:
You should use Url!

Reply:
if it needs another change, I'll drag the implementation into this Century :p